src/java.base/share/classes/java/lang/classfile/constantpool/ConstantPoolBuilder.java

```diff
@@ -474,10 +474,11 @@ default StringEntry stringEntry(String value) {
     }
 
     /**
-     * {@return A {@link ConstantValueEntry} descripbing the provided
+     * {@return A {@link ConstantValueEntry} describing the provided
      * Integer, Long, Float, Double, or String constant}
      *
      * @param c the constant
+     * @see java.lang.classfile.Attributes#constantValue()
```

Comment:
```suggestion
     * @see ConstantValueEntry#constantValue()
```
as it's the reverse operation of this.